README.md

```diff
@@ -83,21 +90,21 @@ await controller.approveToken(contractAddress, spender, amount, signer);
 #### Mint UXD with ETH
 
 ```typescript
-await controller.mintWithEth(market, ethAmount, slippage, signer);
+await controller.mintWithEth(ethAmount, slippage, signer);
```

Comment:
I think we should only showcase `controller.[mint/redeem]` and not `WithEth` variants which could be considered private